aspnetcore-webapi/RemoteHostIpAddress/RemoteHostIpAddress/RemoteHostService.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Net.Sockets;
+using System.Net;
+
+namespace RemoteHostIpAddress
+{
+    public class RemoteHostService : IRemoteHostService
+    {
+        public IPAddress GetRemoteHostIpAddressUsingRemoteIpAddress(HttpContext httpContext)
```

Comment:
If this can return null, we should consider changing the return type to `IPAddress ?`